.vsts-ci/release.yml

```diff
@@ -0,0 +1,274 @@
+    dependsOn: SetPackageVersion
+    variables:
+    - group: ESRP
+    - name: PackageVersion
+      value: $[ dependencies.SetPackageVersion.outputs['Package.Version'] ]
+    pool:
+      name: PowerShell1ES
+      demands:
+      - ImageOverride -equals PSMMS2019-Rust-Secure
+    displayName: Build
+    strategy:
+      matrix:
+        Windows x64:
+          buildName: x86_64-pc-windows-msvc
+        Windows x64_arm64:
+          buildName: aarch64-pc-windows-msvc
+
+    steps:
+    - checkout: self
+    - pwsh: |
+        Set-Location "$(Build.SourcesDirectory)/DSC"
+        ./build.ps1 -Release -Architecture $(buildName) -SkipLinkCheck
+      displayName: 'Build $(buildName)'
+      condition: succeeded()
+    - pwsh: |
+        $null = New-Item -ItemType Directory -Path "$(PackageRoot)" -ErrorAction Ignore
+        $null = New-Item -ItemType Directory -Path "$(PackageRoot)/out" -ErrorAction Ignore
+        $outPath = New-Item -ItemType Directory -Path "$(PackageRoot)/out/$(buildName)" -ErrorAction Ignore
+        Copy-Item -Path "$(Build.SourcesDirectory)/DSC/bin/$(buildName)/$(BuildConfiguration)/*" -Destination $outPath -Verbose -Force
+      displayName: Copy binaries
+      condition: succeeded()
+    - pwsh: |
+        Set-Location "$(PackageRoot)"
+        $signSrcPath = "$(PackageRoot)/out/$(buildName)"
+        # Set signing src path variable
+        $vstsCommandString = "vso[task.setvariable variable=signSrcPath]${signSrcPath}"
+        Write-Host ("sending " + $vstsCommandString)
+        Write-Host "##$vstsCommandString"
+    - pwsh: |
+        $signOutPath = "$(PackageRoot)/signed"
+        $null = New-Item -ItemType Directory -Path $signOutPath
+        # Set signing out path variable
+        $vstsCommandString = "vso[task.setvariable variable=signOutPath;isoutput=true]${signOutPath}"
+        Write-Host "sending " + $vstsCommandString
+        Write-Host "##$vstsCommandString"
+      name: signOutPath
+    - pwsh: |
+        # Set path variable for guardian codesign validation
+        $vstsCommandString = "vso[task.setvariable variable=GDN_CODESIGN_TARGETDIRECTORY]$(signOutPath.signOutPath)"
+        Write-Host "sending " + $vstsCommandString
+        Write-Host "##$vstsCommandString"
+      displayName: Setup variables for signing
+
+    - checkout: ComplianceRepo
+
+    - template: EsrpSign.yml@ComplianceRepo
+      parameters:
+        # the folder which contains the binaries to sign
+        buildOutputPath: $(Build.SourcesDirectory)/DSC/bin/$(buildName)/$(BuildConfiguration)
+        # the location to put the signed output
+        signOutputPath: $(signOutPath.signOutPath)
+        # the certificate ID to use
+        certificateId: "CP-230012"
+        # The file pattern to use
+        # If not using minimatch: comma separated, with * supported
+        # If using minimatch: newline separated, with !, **, and * supported.
+        # See link in the useMinimatch comments.
+        pattern: '*.exe'
```

Comment:
Do we need to sign scripts?

Reply:
Yeah, we should since we have that one.  I'll add